jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -201,6 +201,12 @@
         - shell: JENKINS_BUILD_STARTED=true "${{WORKSPACE}}/_tmp/upload-to-gcs.sh"
         - shell: '{cmd}'
     publishers:
+        - text-finder:
+            regexp: "Test Suite Failed"
+            fileset: donotsearchafile
```

Comment:
yeah, it looks like empty (or not specified) should work:
```
Specify the path to the files in which to search, relative to the workspace root. This can use wildcards like logs/**/*/*.txt. See the @includes of Ant fileset for the exact format. Leave this empty if you do not want to scan any files (usually combined with checked Also search the console output).
```